lib/ea_restaurant_data_loader_web/controllers/oauth2_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule EaRestaurantDataLoaderWeb.Controllers.Oauth2Controller do
+  use EaRestaurantDataLoaderWeb, :controller
+  alias EaRestaurantDataLoader.Lib.Utils.Oauth2Util
+  alias EaRestaurantDataLoader.Lib.Utils.ApplicationUtil
+  alias EaRestaurantDataLoader.Lib.Services.Oauth2Service
+  alias EaRestaurantDataLoader.Lib.ErrorHandlers.BadRequest
+
+  def login(conn, _params) do
+    [client_id, client_secret] =
+      try do
+        [basic_auth | _] = conn |> get_req_header("authorization")
+
+        [client_id, client_secret] =
```

Comment:
Can you remove this `[client_id, client_secret] =`?

Reply:
Done.